lib/src/select/Option.tsx

```diff
@@ -13,41 +13,49 @@ const Option = ({
   isGroupedOption = false,
   isLastOption,
   isSelected,
-}: OptionProps): JSX.Element => (
-  <OptionItem
-    id={id}
-    onClick={() => {
-      onClick(option);
-    }}
-    visualFocused={visualFocused}
-    selected={isSelected}
-    role="option"
-    aria-selected={isSelected}
-  >
-    <StyledOption
+}: OptionProps): JSX.Element => {
+  const labelRef = useRef<HTMLSpanElement>(null);
+
+  useEffect(() => {
+    if (labelRef.current.scrollWidth > labelRef.current.clientWidth) document.getElementById(id).title = option.label;
```

Comment:
The first time opening a select labelRef.current.clientWidth can be 0.

Reply:
The `useEffect` executes after the component is rendered, how could that be possible?

Reply:
![image](https://github.com/dxc-technology/halstack-react/assets/12238686/03d41d5d-924c-40bb-9b4b-05dbe83a1d83)
![image](https://github.com/dxc-technology/halstack-react/assets/12238686/97186789-6cd0-4efc-98d7-c73ba2feaba2)
These are screenshot from the first opening.

![image](https://github.com/dxc-technology/halstack-react/assets/12238686/f2dc9ce2-ba69-46e0-8631-a6e14900b12b)
And this one after closing and reopening the list.